### DISCUSSION TITLE
Generating a Preview Image in PrusaSlicer using the CreateHeaderAndThumbnail.py

### DISCUSSION THREAD

**Discussion by ilovehatersrawraw** [mriscoc/Ender3V2S1#782] [Q&A]
Hello everyone, I hope all is well!!  Here is the situation I am encountering.

I get this error code when exporting the job to the sd card. 
 
Error code: 1 Output: File "/home/raw/Prusa/PrusaSlicerScripts/CreateHeaderAndThumbnail.py", line 72 <title>Ender3V2S1/CreateHeaderAndThumbnail.py at Ender3V2S1-Released · mriscoc/Ender3V2S1</title> ^ SyntaxError: invalid character '·' (U+00B7

This is for an Ender V2 Neo with the Professional Firmware.
I have tried deleting the bullet in text editor but that didn't help

**Comment by mriscoc** [comment]
Follow the Wiki guide and move the script to a shorter path without symbols or spaces.

**Reply by ilovehatersrawraw** [reply]
Ok got it!  thank you sir!!!
